mws/resource_mws_credentials.go

```diff
@@ -59,71 +29,82 @@ func (a CredentialsAPI) List(mwsAcctID string) ([]Credentials, error) {
 	return mwsCredsList, err
 }
 
+type CredentialInfo struct {
```

Comment:
I see that you are creating this because the one in the SDK does not match the TF. But this also means that we still need to maintain this one by hand (e.g., when a new field is added). Is there anyway we could improve that?

Reply:
this would need a more generic function to combine the `CreateCredentialRequest` struct and `Credential` for the responses. 

Additionally, this was a very old resource, so the tf schema flattens the struct `AwsCredentials`. Any changes within this will need maintenance by hand.